packages/sandbox/ui/src/components/component-controls/print-value.ts

```diff
@@ -23,7 +23,7 @@ export const printValue = (value?: ReactElement<any> | string | number | symbol)
   }
 
   if (isFunction(value)) {
-    return '{() => {}}'
+    return value.displayName ? value.displayName : (value.name ? value.name : '{() => {}}')
```

Comment:
LoL what is this github action mumbo jumbo "do not nest ternary" YOU'RE NOT MY REAL MOM

Reply:
oof

Reply:
fix 🕺